src/main.rs

```diff
@@ -21,6 +23,8 @@ async fn main() -> Result<()> {
 
     // mostro pubkey
     let pubkey = var("MOSTRO_PUBKEY").expect("$MOSTRO_PUBKEY env var needs to be set");
+    //Used to get upper currency string to check against a list of tickers
```

Comment:
for readability is better separate the word from the slash, I mean

`//Used to get` => `// Used to get`